cli/studio.py

```diff
@@ -187,58 +187,7 @@ def replay(directory: Sequence[str], timestep: float, endpoint: str):
             )
 
 
-@scenario_cli.command(
-    name="browse-waymo",
```

Comment:
The command `scl scenario browse-waymo` was referred to in https://github.com/huawei-noah/SMARTS/tree/saul/waymo-utils/smarts/waymo#running-the-utility . 

What should the equivalent new explanation be in the readme or should the scl command example be removed from the readme?

Reply:
I removed it from `scl` to keep it distinct from the newly added `scl waymo` commands. I've now removed that part from the README.